source/docs/advanced/payments/hipay.md

```diff
@@ -0,0 +1,186 @@
+---
+id: hipay
+title: HiPay integration
+---
+
+This page contains information about the different ways you can accept payments with [HiPay](https://hipay.com/) in your Front-Commerce application.
+
+<blockquote class="feature--new">
+  _Since version 2.13.0_
+</blockquote>
+
+## Configuring the HiPay console
+
+For Front-Commerce to use the HiPay API you will need to create technical users and retrieve some informations for the environment variables.
+
+- Go to Menu > Integration > Security Settings.
+- Get the "Secret Passphrase" as `FRONT_COMMERCE_HIPAY_SECRET_PASSPHRASE`
+- Ensure the "Hashing Algorithm" is SHA-256
+- Create the public user
+  - Clik on "Generate new credentials"
+  - Set "Credentials accessibility" to "Public"
+  - Check "Tokenize a card"
+  - Click on "save"
+  - Get the "username" as `FRONT_COMMERCE_HIPAY_PUBLIC_USERNAME`
+  - Get the "password" as `FRONT_COMMERCE_HIPAY_PUBLIC_PASSWORD`
+- Create the private user
+  - Clik on "Generate new credentials"
+  - Set "Credentials accessibility" to "Private"
+  - Check all in "Order" and "Maintenance"
+  - Click on "save"
+  - Get the "username" as `FRONT_COMMERCE_HIPAY_PRIVATE_USERNAME`
+  - Get the "password" as `FRONT_COMMERCE_HIPAY_PRIVATE_PASSWORD`
+
+### Add the HiPay settings to the environment
+
+You need to define several environment variables to enable HiPay payments (remember you can fix the in the `.env` file)
```

Comment:
**nitpick:** I don't think it brings any value here to mention how to define env vars.\
Integrators will already have to figure this out for other configurations in a FC project.

**suggestion:** simplify the sentence
```suggestion
You need to define several environment variables to enable HiPay payments:
```